test/Cpp/src/PhysicalConnection.lf

```diff
@@ -0,0 +1,31 @@
+/**
+*
+* @author Maiko Brants TU Dresden
+*
+* Test physical connections.
+*
+* Modeled after the C version of this test.
+**/
+target Cpp;
+reactor Source {
+    output out:int;
+    reaction(startup) -> out {=
+        out.set(42);
+    =}
+}
+reactor Destination {
+    input in:int;
+    reaction(in) {=
+        auto time{get_elapsed_physical_time()};
```

Comment:
Shouldn't this be *logical* time?

Reply:
I think I copied the the wrong error text here. This is the PhysicalConnection Test so the physical time is correct I think.